app/presenters/idv/forgot_password_presenter.rb

```diff
@@ -0,0 +1,35 @@
+module Idv
+  class ForgotPasswordPresenter < FailurePresenter
```

Comment:
What's the reason for using the failure presenter here and not adding another view? We've already had some frustration around the failure presenter class being re-used too much and this doesn't really seem like a failure state.

Reply:
It's already used for warnings:     warning: {
      icon: 'alert/warning-lg.svg',
      alt_text: 'warning',
      color: 'yellow',
    },

Reply:
Yeah, I gotcha. What I'm remembering is when we wanted to change some text on the MFA warning page, but changing the template for that resulted it being changed everywhere else we used the failure presenter. I worry extending it further will exacerbate that. The call is yours though.